lib/muffin_man/v20201201/catalog_items.rb

```diff
@@ -0,0 +1,38 @@
+module MuffinMan
+  module V20201201
```

Comment:
@natesalisbury we decided to scope versions of the API as follows

Reply:
I think that makes sense. We should do a separate PR to move existing APIs to a version namespace (currently only Solicitations and Reports). Then do a major version bump since it wouldn't be backwards compatible.